solidity/dashboard/src/pages/ApplicationsPageContainer.jsx

```diff
@@ -1,25 +1,30 @@
 import React from "react"
 import { Route, Switch, Redirect } from "react-router-dom"
-import ApplicationOverviewPage from "./ApplicationOverviewPage"
 import TBTCApplicationPage from "./TBTCApplicationPage"
 import KeepRandomBeaconApplicationPage from "./KeepRandomBeaconApplicationPage"
+import PageWrapper from "../components/PageWrapper"
 
+const subLinks = [
+  { title: "Random Beacon", path: "/applications/random-beacon" },
+  { title: "tBTC", path: "/applications/tbtc" },
+]
 const ApplicationsPageContainer = (props) => {
   return (
-    <Switch>
-      <Route
-        exact
-        path="/applications/overview"
-        component={ApplicationOverviewPage}
-      />
-      <Route exact path="/applications/tbtc" component={TBTCApplicationPage} />
-      <Route
-        exact
-        path="/applications/random-beacon"
-        component={KeepRandomBeaconApplicationPage}
-      />
-      <Redirect to="/applications/overview" />
-    </Switch>
+    <PageWrapper title="Applications" subLinks={subLinks}>
+      <Switch>
+        <Route
+          exact
+          path="/applications/tbtc"
+          component={TBTCApplicationPage}
+        />
+        <Route
+          exact
+          path="/applications/random-beacon"
+          component={KeepRandomBeaconApplicationPage}
+        />
+        <Redirect to="/applications/random-beacon" />
```

Comment:
This is the “if nothing else matches” fallthrough?

Reply:
yeah

> You can use the last `Route` in a `Switch` as a kind of "fallback" route, to catch 404 errors. A `Switch` renders the first child `Route` that matches

^ from the `react-router` docs